src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -791,8 +791,10 @@ private void addNewResponses(
         }
 
         StudentAttributes student = studentsLogic.getStudentForEmail(question.courseId, studentEmail);
-        if (question.recipientType.isTeam()
```

Comment:
I think we have to remove this `isTeam` method on FeedbackParticipantType since a CUSTOM type may contain a team.